docs/source/en/training/lora.mdx

```diff
@@ -150,6 +150,27 @@ This is especially useful when you don't want to hardcode the base model identif
 Inference for DreamBooth training remains the same. Check
 [this section](https://github.com/huggingface/diffusers/tree/main/examples/dreambooth#inference-1) for more details. 
 
+### Merging LoRA with original model
+
+You can adjust the merging ratio between the fine-tuned model and the original. This is controlled by a parameter called α (alpha) in the paper, or `scale` in our implementation. You can tweak it with the following code, that passes `scale` as `cross_attention_kwargs` in the pipeline call:
+
```

Comment:
Maybe add a preamble stating that during inference, LoRA weights are merged with the frozen pre-trained weights and then give a small example?

Reply:
Well, this is inside the inference section already :) I thought that all references to "merging" were clear, but I'll reword a little bit. I'm not sure what you mean by a "small example", do you mean an image strip?

Reply:
Yeah, maybe an image strip showing two weight matrices being merged (I think the LoRA paper already has something similar). Or maybe an equation to make it explicitly clear. No strong opinions, though.